verification/curator-service/api/src/model/source.ts

```diff
@@ -49,10 +49,7 @@ interface Automation {
 
 const automationSchema = new mongoose.Schema({
     name: {
-        name: {
```

Comment:
removed the required part here as it was a forgotten comment from Alex in a previous PR, now done.